lessons/pydata/classification_metrics/index.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>Jedná se o ..., kterou ve třicátých letech sestavil statistik a biolog Ronald Fisher (<a href="https://en.wikipedia.org/wiki/Iris_flower_data_set" target="_blank">https://en.wikipedia.org/wiki/Iris_flower_data_set</a>) ?</p><p> </p><p>květů (amerických) kosT[sic]atců, taky bych možná psal kategorie biologicky správně s malým písmenem.</p><p> </p><p>kališT[sic]ních (sepal) a okvětních (petal) - aby to nepůsobilo tak mysteriózně</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>